lint-tasks.yml

```diff
@@ -42,6 +47,34 @@ tasks:
           FLAGS: "-i"
     sources: *cpp_source_files
 
+  js-check:
+    dir: "{{.TASKFILE_DIR}}/components/webui"
+    cmds:
+      - task: "js"
+        vars:
+          LINT_CMD: "check"
+    sources: &js_source_files
+      - "{{.TASKFILE_DIR}}/lint-tasks.yml"
+      - "{{.TASKFILE_DIR}}/Taskfile.yml"
```

Comment:
Rather than making this task depend on the root Taskfile (which isn't a direct dependency, right?), I think it makes more sense to make it depend on a checksum of the node modules?